HW03-VaughtK/hw03_VaughtK.py

```diff
@@ -0,0 +1,182 @@
+
+# TODO Do not edit this file directly!
+# TODO Instead, create a new file called hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Kai Vaught
+# Username: VaughtK
+#
+# Assignment: Homework 3
+# Purpose: To learn how branches work.
+# Google Doc Link: https://docs.google.com/document/d/1OvsmR456y61n3UzwBlIlVE13IBz0tF39Vyv10F24ZGI/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+
+def house_turtle():
```

Comment:
Typically we use verbs for function names, since they are actions/do things. I would suggest renaming to `def draw_house():`